x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_schema.tsx

```diff
@@ -300,28 +359,116 @@ export const EngineSchema: React.FC = () => {
+              <EuiFlexItem grow={false}>
+                {i18n.translate('xpack.enterpriseSearch.content.engine.schema.filters.label', {
+                  defaultMessage: 'Filter By',
+                })}
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiPopover
+                  button={filterButton}
+                  isOpen={isFilterByPopoverOpen}
+                  closePopover={() => setIsFilterByPopoverOpen(false)}
+                  panelPaddingSize="none"
+                  anchorPosition="downCenter"
+                >
+                  <EuiSelectable
+                    searchable
+                    searchProps={{
+                      placeholder: i18n.translate(
+                        'xpack.enterpriseSearch.content.engine.schema.filters.searchPlaceholder',
+                        {
+                          defaultMessage: 'Filter list ',
+                        }
+                      ),
+                    }}
+                    options={selectedEsFieldTypes}
+                    onChange={(options) => setSelectedEsFieldTypes(options)}
+                  >
+                    {(list, search) => (
+                      <div style={{ width: 300 }}>
+                        <EuiPopoverTitle paddingSize="s">{search}</EuiPopoverTitle>
+                        {list}
+                      </div>
+                    )}
+                  </EuiSelectable>
+                  <EuiPopoverFooter>
+                    <EuiFlexGroup justifyContent="spaceAround">
+                      <EuiButtonEmpty
+                        color="danger"
+                        iconType="eraser"
+                        size="s"
+                        onClick={() => setSelectedEsFieldTypes(esFieldTypes)}
+                      >
+                        {i18n.translate(
+                          'xpack.enterpriseSearch.content.engine.schema.filters.clearAll',
+                          {
+                            defaultMessage: 'Clear all ',
+                          }
+                        )}
+                      </EuiButtonEmpty>
+                    </EuiFlexGroup>
+                  </EuiPopoverFooter>
+                </EuiPopover>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexGroup>
+
+          <EuiBasicTable
+            items={filteredSchemaFields}
+            columns={columns}
+            loading={isLoadingEngineSchema}
+            itemId="name"
+            itemIdToExpandedRowMap={itemIdToExpandedRowMap}
+            isExpandable
           />
+          {totalFieldsWithConflicts > 0 && (
+            <EuiCallOut
+              title={
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.content.engine.schema.filters.conflict.callout.title"
+                  defaultMessage="There are {totalFieldsWithConflicts} more conflicts not displayed here"
+                  values={{ totalFieldsWithConflicts }}
```

Comment:
we should handle the pluralization here so this message is

`There is one more conflict not displayed here` when there's only `1` hidden conflict or

`There are {n} more conflicts not displayed here` otherwise.

this `{plural}` formatting clause makes that easy: https://formatjs.io/docs/core-concepts/icu-syntax#plural-format